### PR TITLE
Fix workflow translations with unicode characters

### PR DESCRIPTION
**Pull request by Gagaro**
I added a workflow in the testing profile (`test_workflow`) which is the `simple_publication_workflow` but with a unicode character in every title. I set it as the workflow for the custom testing types.

### CODE REVIEW

**Comment by tisto** [comment]
@Gagaro would it be possible to get away without adding a full custom workflow to the testing profile? For instance just amending an existing workflow via Python code?

I really dislike testing profiles (not your fault of course) because they represent a global state in the test that is hard to maintain and easy to break. I wanted to get rid of the testing profile all together for quite some time and merging this PR would make that a lot harder...

**Comment by Gagaro** [comment]
The thing I like by adding the workflow in the profile is that the unicode character will be tested everywhere instead of just one test.

I'll see how I can change a specific workflow to have that in a single test if you prefer though.

**Comment by tisto** [comment]
@Gagaro right, I forgot about the change in the HTTP status. Then it's actually a breaking change.

**Comment by tisto** [comment]
@Gagaro true. Though, don't you think we are testing special characters on the workflow on the wrong level then? Unit/integration tests in plone.restapi should only test plone.restapi functionality, not Plone functionality.

**Comment by Gagaro** [comment]
This bug is in plone.restapi. It could be somewhere else than `WorkflowInfo` and we can't be sure unless we test it everywhere.

**Comment by tisto** [comment]
@Gagaro what about just amending the default workflow used in the test fixture to use a special character?  This would allow us to reduce the full testing profile with two or three lines of Python code which would make the test fixture a lot more readable and maintainable. What do you think?

**Comment by Gagaro** [comment]
I did that and rebased on master.

**Comment by tisto** [comment]
@Gagaro awesome! Thank you!

**Comment by tdesvenain** [comment]
Hi i also have to include a commit that fixes the @transition endpoint https://github.com/plone/plone.restapi/commit/b1c035e8ff39592a6c4b61f1992ae635b86b6ade

**Comment by tisto** [comment]
@tdesvenain now I am confused. Shall I wait with merging this one?

**Comment by tdesvenain** [comment]
@tisto yes, sorry for the confusion
and this was this commit : https://github.com/makinacorpus/plone.restapi/commit/88ca28bf2c467f0f4167ef45c1e1958415dd910c

**Comment by Gagaro** [comment]
Yes, I just added the fix + the updated test.

**Comment by tisto** [comment]
Ok. Cool. Then you might want to update the changelog.

File: CHANGES.rst [hunk 1]
```diff
@@ -7,6 +7,10 @@ Changelog
 - Build documentation on Plone 5.0.x (before: Plone 4.3.x).
   [timo]
 
+- Fix workflow translations with unicode characters.
```
Comment (tisto) [inline CHANGES.rst]: @Gagaro this one is missing an author.

Comment (Gagaro) [reply CHANGES.rst]: It's the same for both. I added it.

**Comment by tisto** [comment]
@Gagaro Travis build fails. Please fix code violations.

**Comment by tisto** [comment]
@Gagaro ping :)

**Comment by Gagaro** [comment]
Sorry for the delay, it's all green now.

**Comment by tisto** [comment]
Thanks a lot @Gagaro! It took a while but I think the result was worth the effort. :)

**Pull request merged**